### PR TITLE
Docs changes

### PR DESCRIPTION
**Pull request by kkellerlbl**
Various docs changes for doing local deployments and testing.  Github is saying there's a conflict, but I'm not sure where.  I'll go through part of the manual merge and see what I can find.

### CODE REVIEW

**Comment by kkellerlbl** [comment]
The conflict was extremely minor.  Let me see if I can fix the PR to clear it.

**Comment by kkellerlbl** [comment]
My try didn't work.  Can someone review my changes and either merge by hand, or let me know they're okay and I will merge by hand?

File: doc/Module_builder.md [hunk 1]
```diff
@@ -13,13 +13,14 @@ An example KBase service written in Java will be available soon.
...
-1. MacOS 8 or higher.
+1. Mac: OS X 10.8 or higher.
  - The SDK should also work on Linux but has not yet been extensively tested. We are not yet supporting Windows.
 2. Java JDK (version 7 or above): http://java.oracle.com/
  - You will also need to set the JAVA_HOME environment variable to point to your Java installation.
 3. Latest version of XCode (includes make and other developer tools)
-4. ant: http://java.oracle.com/
-5. git (command-line version): https://git-scm.com/
+4. ant: http://ant.apache.org/
+ - You will also need to add ant to your PATH environment variable.
```
Comment (nlharris) [inline doc/Module_builder.md]: "You will also need to add ant to your PATH environment variable." -- this is almost right. You need to add to your PATH the *directory* that the ant executable lives in. I had put this instruction later:
> Also, be sure your PATH includes the directory that has the ant executable.

(Thanks for catching the mis-paste of the ant URL, though! Oops!)

File: doc/Module_builder.md [hunk 2]
```diff
@@ -13,13 +13,14 @@ An example KBase service written in Java will be available soon.
...
+5. git (command-line version): https://git-scm.com/ (should come with Xcode)
```
Comment (nlharris) [inline doc/Module_builder.md]: This sounds like you're saying that if you get git, you also get Xcode, when in fact it's the opposite. Maybe say something like "Install Xcode (URL) to get this"?

**Comment by nlharris** [comment]
Your changes look fine. I don't see where the conflicts are. Is there a view that shows the conflicts?

**Comment by kkellerlbl** [comment]
Thanks Nomi.  I just added your suggestions to the branch.

Unfortunately the actual conflicts are not obvious in the side-by-side view; at least, I haven't found them to be.  I've had to go to the command line to see the actual conflicts.  In this case I think the "conflict" is actually bogus but I think the only way to get around it is to use the command line to do the merge.  I can do that if my latest changes look okay.

**Comment by kkellerlbl** [comment]
I merged my changes by hand.  The ''conflict'' was some extra text I added that really shouldn't have resulted in a conflict.

**Pull request merged**